templates/graph/default_sidebar.html

```diff
@@ -48,6 +48,11 @@
                 Set as <br>default layout
             </a>
         </li>
+        <li>
+            <a id="setDefaultAutoLayoutBtn" class="btn" href="#" style="display: none" onclick="graphPage.defaultLayoutWidget.onSetDefaultAutoLayoutBtn()">
```

Comment:
@lohani2280 Why do we need a separate button to handle auto layouts? Cant we just check the auto_layout query parameter and call the appropriate function?

Reply:
@adbharadwaj Sorry, my bad. You are right, we don't need a separate button. I'll update the PR with this change soon.